stbt/__init__.py

```diff
@@ -39,6 +39,11 @@
     UITestError, \
     UITestFailure, \
     wait_until
+from _stbt.transition import \
+    TransitionResult, \
```

Comment:
I don't think `TransitionResult` belongs here.  Users should not be constructing them.